### PR TITLE
ASoC: SOF: skl Topic/sof dev enable skl

### PR DESCRIPTION
**Pull request by zhuyingjiang**
enable the skl/kbl ops, instead of using apl_ops

### CODE REVIEW

File: sound/soc/sof/intel/hda-dsp.c [hunk 1]
```diff
@@ -208,6 +208,21 @@ bool hda_dsp_core_is_enabled(struct snd_sof_dev *sdev,
 	return is_enable;
 }
 
+int hda_dsp_enable_core(struct snd_sof_dev *sdev, unsigned int core_mask)
```
Comment (lgirdwood) [inline sound/soc/sof/intel/hda-dsp.c:211]: Generally f you have to number features in a patch commit message it means the patch needs split into more smaller patches

**Comment by zhuyingjiang** [comment]
@lgirdwood is it OK for merge the first two patch?

File: sound/soc/sof/intel/hda-loader.c [hunk 2]
```diff
@@ -372,3 +372,156 @@ int hda_dsp_cl_boot_firmware(struct snd_sof_dev *sdev)
...
+err:
+	hda_dsp_dump(sdev, SOF_DBG_REGS | SOF_DBG_PCI | SOF_DBG_MBOX);
+	hda_dsp_core_reset_power_down(sdev, HDA_DSP_CORE_MASK(0));
+out:
+	return ret;
+}
+
+static int cl_copy_fw_skl(struct snd_sof_dev *sdev)
+{
+	return -1;
```
Comment (lgirdwood) [inline sound/soc/sof/intel/hda-loader.c:472]: This needs removed for merge

File: sound/soc/sof/intel/hda-loader.c [hunk 3]
```diff
@@ -372,3 +372,156 @@ int hda_dsp_cl_boot_firmware(struct snd_sof_dev *sdev)
...
+	ret = snd_sof_dsp_register_poll(sdev, HDA_DSP_BAR,
+					HDA_DSP_SRAM_REG_ROM_STATUS_SKL,
+					HDA_DSP_ROM_STS_MASK, HDA_DSP_ROM_INIT,
+					HDA_DSP_INIT_TIMEOUT);
+	if (ret >= 0) {
+		dev_err(sdev->dev, "error: can't read the ROM status!");
+		goto out;
+	}
+
+	ret = -EIO;
```
Comment (lgirdwood) [inline sound/soc/sof/intel/hda-loader.c:461]: Shouldn't we return 0 here for success ?

**Comment by lgirdwood** [comment]
@zhuyingjiang generally all debug must be removed before merge. Two other comments.

**Pull request closed without merging**